benchmark/main.cpp

```diff
@@ -203,20 +222,23 @@ int main(int argc, char* argv[]) {
         useInt64 = (find(result.begin(), result.end(), "64") != result.end());
     }
 
-    cout << "Column size in KB,Data type,Time in ns,Cache,Thread Count" << endl;
+    cout << "Column size in KB,Data type,Time in ns,Cache,Thread Count,DB type" << endl;
     for (auto size: DB_SIZES){
         cerr << "benchmarking " << (size / 1024.0f) << " KiB" << endl;
+
+        int mIterations = iterations == -1 ? max(1, (int) (1 / ITERATIONS_FACTOR / size * 8 * KiB)) : iterations;
```

Comment:
#Insert useful comment here and intuition behind
int mIterations = iterations ? max(1, (int) (1 / ITERATIONS_FACTOR / size * 8 * KiB)) : iterations;